### ISSUE TITLE
Typo in `no-negation-in-equality-check`'s message

### ISSUE THREAD

**Issue by incendial** [sindresorhus/eslint-plugin-unicorn#2483]
Hi, it looks like there is a typo in `no-negation-in-equality-check`'s message https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/rules/no-negation-in-equality-check.js#L15

```js
const messages = {
  [MESSAGE_ID_ERROR]: 'Negated expression in not allowed in equality check.', // Should be "is not" instead of "in not"
  [MESSAGE_ID_SUGGESTION]: 'Switch to \'{{operator}}\' check.',
};
```

**Comment by fregante** [comment]
@incendial Thanks! Fixed
